### PR TITLE
optimize PDF export UI

### PR DESCRIPTION
**Pull request by alsturm**

### CODE REVIEW

File: src/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.js [hunk 1]
```diff
@@ -2,6 +2,9 @@ import { html } from 'lit-html';
 import { $injector } from '../../../../injection';
 import { AbstractToolContent } from '../toolContainer/AbstractToolContent';
 import { cancelJob, requestJob, setId, setScale } from '../../../../store/mfp/mfp.action';
+import css from './exportMfpToolContent.css';
+import plusSvg from './assets/plusSvg.svg';
+import minusSvg from './assets/minusSvg.svg';
```
Comment (thiloSchlemmer) [inline src/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.js:7]: Redundant extension info. How about `import plusSvg from './assets/plus.svg';`

**Review by thiloSchlemmer — APPROVED** [review]
Nice!

File: src/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.js (hunk 1 above)
Comment (alsturm) [reply src/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.js]: done

File: src/modules/toolbox/i18n/toolbox.provider.js [hunk 2]
```diff
@@ -91,6 +91,8 @@ export const provide = (lang) => {
 				toolbox_exportMfp_header: 'Export to PDF',
 				toolbox_exportMfp_layout: 'Page layout',
 				toolbox_exportMfp_scale: 'Scale',
+				toolbox_exportMfp_scale_decrease: 'Scale decrease',
+				toolbox_exportMfp_scale_increase: 'Scale increase',
```
Comment (taulinger) [inline src/modules/toolbox/i18n/toolbox.provider.js:95]: "Decrease scale"

"Increase scale"

File: src/modules/toolbox/i18n/toolbox.provider.js [hunk 3]
```diff
@@ -91,6 +91,8 @@ export const provide = (lang) => {
 				toolbox_exportMfp_header: 'Export to PDF',
 				toolbox_exportMfp_layout: 'Page layout',
```
Comment (taulinger) [inline src/modules/toolbox/i18n/toolbox.provider.js:92]: just "Layout" ?

File: test/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.test.js [hunk 4]
```diff
@@ -95,7 +95,7 @@ describe('ExportMfpToolContent', () => {
 			spyOn(mfpServiceMock, 'getCapabilities').and.returnValue(capabilities);
 			const element = await setup({ ...mfpDefaultState, current: initialCurrent });
 
-			expect(element.shadowRoot.querySelector('#select_layout')).toBeTruthy();
+			expect(element.shadowRoot.querySelector('.layout-button')).toBeTruthy();
```
Comment (taulinger) [inline test/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.test.js:98]: better `expect(element.shadowRoot.querySelectorAll('.layout-button')).toHaveSize(1);`

Let's try to remove all truthy/falsy tests here...

File: test/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.test.js [hunk 5]
```diff
@@ -115,36 +115,36 @@ describe('ExportMfpToolContent', () => {
 			spyOn(mfpServiceMock, 'getCapabilities').and.returnValue(capabilities);
 			const element = await setup({ ...mfpDefaultState, current: initialCurrent });
 
-			expect(element.shadowRoot.querySelectorAll('#select_layout option')).toHaveSize(2);
+			expect(element.shadowRoot.querySelectorAll('.layout-button')).toHaveSize(2);
 			expect(element.shadowRoot.querySelectorAll('#select_scale option')).toHaveSize(3);
```
Comment (taulinger) [inline test/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.test.js:119]: Would expect here also a test for the presence of the active css class on the correct position

**Review by taulinger — APPROVED** [review]
Bring it on!

**Pull request merged**